app/src/main/res/drawable/rounded_line.xml

```diff
@@ -0,0 +1,27 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
+        <shape>
+            <solid android:color="@color/ocean" />
+            <corners android:radius="5dp" />
+            <size android:height="5dp" />
+            <!-- 50% opacity @color/asphalt -->
```

Comment:
This line was accidentally left there, it can be removed.

Reply:
oh true